### DISCUSSION TITLE
server-side

### DISCUSSION THREAD

**Discussion by Kurotsune77** [vlad2305m/Sound-Physics-Fabric#72] [Q&A]
Does Sound Physics do anything server-side if I install it on the server, like I'm most curious about whether it adds anything to the client-server communication, but I guess this could also serve as a general "Is there any benefit to installing Sound Physics server side?" kinda question

**Comment by Vut0r** [comment]
This is a very good question, I came here to check if this is something that has been addressed but it seems that your doubt was left unresolved. I'll do a bit testing on my own server and let you know in case you havnen't figure it out yet.

**Reply by Vut0r** [reply]
Quick updates: It is requiered. It is not Client only.
